pkg/controller/secret/secret_controller.go

```diff
@@ -155,22 +155,22 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 	err = r.client.Get(context.TODO(), kubeTypes.NamespacedName{Name: userDataSecret, Namespace: userDataNamespace}, userData)
 	if err != nil && k8sapierrors.IsNotFound(err) {
 		// Secret is deleted
-		reqLogger.Info("UserData secret not found, creating the secret.")
+		log.Info("windows-user-data secret not found, creating the secret")
```

Comment:
you can use `secrets.userDataSecret` here

Reply:
applies to all uses in the file

Reply:
I thought about this but the log becomes a bit wordy and does not read well. Are you okay with `log.Info("secret not found, creating the secret", "name", userDataSecret)`?

Reply:
Sounds good to me